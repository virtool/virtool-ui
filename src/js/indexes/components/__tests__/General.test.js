import IndexGeneral from "../General";

describe("<IndexGeneral />", () => {
    it("should render", () => {
        const wrapper = shallow(<IndexGeneral />);
        expect(wrapper).toMatchSnapshot();
    });
});
