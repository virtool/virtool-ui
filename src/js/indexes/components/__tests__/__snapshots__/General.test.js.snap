// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IndexGeneral /> should render 1`] = `
<div>
  <Connect(Contributors) />
  <Connect(IndexOTUs) />
</div>
`;
